### PR TITLE
Feat/tips and tricks

### PR DESCRIPTION
**Pull request by paulywill**
There were a few things mentioned real time during the v.5 meetup regarding git.

I thought I would throw them in a documented folder while they're still fresh in my mind.

### CODE REVIEW

**Comment by T357A** [comment]
How do I find what has been assigned to me? :D

File: README.md [hunk 1]
```diff
@@ -1,29 +1,79 @@
...
+
+    `git clone https://github.com/[your_username]/rubberduck.git`
+3. Install node packages:
+
+    `cd backend`
+
+    `npm install`
+
+4. Create MongoDB, we're using an online service (free tier) [MongoDB Atlas](https://www.mongodb.com/cloud/atlas). If you want the same one we're using just ask one of us for the secret credentials (just not [@paulywill's](https://github.com/paulywill) credit card  [@T357A](https://github.com/T357A)!!!)
+
```
Comment (T357A) [inline README.md:52]: LOL

**Review by T357A — COMMENTED** [review]
Viewed files (T357A)

File: README.md (hunk 1 above)
Comment (paulywill) [reply README.md:52]: Thought you'd like that.;)

**Pull request merged**